backend/tracim_backend/lib/core/workspace.py

```diff
@@ -216,12 +216,14 @@ def enable_notifications(self, user: User, workspace: Workspace):
             if role.workspace == workspace:
                 role.do_notify = True
 
-    def get_notifiable_roles(self, workspace: Workspace) -> [UserRoleInWorkspace]:
+    def get_notifiable_roles(
+        self, workspace: Workspace, force_notify: bool = False
+    ) -> [UserRoleInWorkspace]:
         roles = []
         for role in workspace.roles:
             if (
-                role.do_notify is True
-                and role.user != self._user
+                (force_notify or role.do_notify is True)
+                and (not self._user or role.user != self._user)
                 and role.user.is_active
                 and not role.user.is_deleted
                 and role.user.auth_type != AuthType.UNKNOWN
```

Comment:
Technical debt issue: These conditions can be stored in unique function somewhere to improve maintainability.

Reply:
issue there  #2208

Reply:
issue related there #2208